apps/transport/lib/transport_web/templates/stats/index.html.eex

```diff
@@ -118,6 +118,32 @@
         </div>
       </div>
     </div>
+
+    <div class="domain bikes">
+      <h2>Vélos libre service</h2>
+      <div class="domain-stats">
+        <%= render("_maps.html", droms: @droms, prefix: "bikes_map") %>
+        <div class="panel">
+          <div class="description">
+            <p> La carte des données disponibles de vélos en libre service.
```

Comment:
pour le texte je propose de reprendre un peu ce qu'il y a dans la [doc](https://doc.transport.data.gouv.fr/producteurs/velos-en-libre-service):
```markdown
La plateforme transport.data.gouv.fr référence des données temps réel de vélos en libre service (VLS) au format [GBFS](https://github.com/NABSA/gbfs). Le GBFS est le standard ouvert pour les VLS et des services en free-floating.
Les données, rafraîchies toutes les minutes, permettent de connaître:  
* La liste des stations du réseau (géolocalisation, nom, adresse, identifiant) ;
* L'état des stations (nombre de vélos disponibles et emplacements libres). 

Plus d’informations sur les données temps réel de vélos en libre service sont disponibles dans la [documentation](https://doc.transport.data.gouv.fr/producteurs/velos-en-libre-service).
```

Tu en penses quoi @be-mercier ?

Reply:
Top! Ca me va bien ! 
Je peux proposer une légère modif du texte : 

`La plateforme transport.data.gouv.fr référence des données de disponibilité en temps réel de vélos en libre service (VLS) au format [GBFS](https://github.com/NABSA/gbfs). Le GBFS est le standard ouvert pour les VLS et des services en free-floating.
Les données, rafraîchies toutes les minutes, permettent de connaître:  
* La liste des stations du réseau (géolocalisation, nom, adresse, identifiant) ;
* L'état des stations (nombre de vélos disponibles et emplacements libres). 

Plus d’informations sur les données temps réel de vélos en libre service sont disponibles dans la [documentation](https://doc.transport.data.gouv.fr/producteurs/velos-en-libre-service).`